roadmap-reviews/2020/review_3_june_20.md

```diff
@@ -0,0 +1,134 @@
+- Revisit exception taxonomy
+  ([#126](https://github.com/in-toto/in-toto/issues/126)) -- *the current
+  taxonomy is now thoroughly documented via
+  [#369](https://github.com/in-toto/in-toto/pull/369) but the issue is kept to
+  motivate future enhancements as part of the roadmap '21*
+- Verify in-toto's supply chain with in-toto
+  ([#278](https://github.com/in-toto/in-toto/issues/278)) -- *will be carried
+  over to the roadmap '21*
+- Parametrize writing of inspection links
+  ([#260](https://github.com/in-toto/in-toto/issues/260)) -- *no more roadmap
+  priority, also see [#364](https://github.com/in-toto/in-toto/pull/364) for
+  important preparatory work for the issue*
+- Require fully conformant metadata when loading
+  ([#186](https://github.com/in-toto/in-toto/issues/186)) -- *no more roadmap
+  priority*
+- Integrate/require inspection commands
+  ([#109](https://github.com/in-toto/in-toto/issues/109)) -- *no more roadmap
+  priority*
+- Revisit artifact rule path patterns
+  ([#32](https://github.com/in-toto/docs/issues/32)) -- *was transferred to
+  specification issue tracker*
+
+
+We are working hard to make 1.0 become a reality early in the next roadmap.
+
+
+
+## Preparing 0.5.0
+
+We are preparing this evaluation period's pre-release, which includes a bug
+fix, a new feature and clean-up work in the unit test suite.
+
+### Relevant documents
+
+- Bugfix: Use kwargs in in-toto-run to fix lstrip-paths bug
+  ([#340](https://github.com/in-toto/in-toto/pull/340))
+- Feature: Add a flag to accept a target directory to put generated metadata
+  ([#364](https://github.com/in-toto/in-toto/pull/364))
+- Tests: Add tmp dir and gpg key test mixins
+  ([#345](https://github.com/in-toto/in-toto/pull/345))
+- Tests: Use constant from securesystemslib to detect GPG in tests
+  ([#352](https://github.com/in-toto/in-toto/pull/352))
+- Tests: Enhance test suite feedback on Windows
+  ([#368](https://github.com/in-toto/in-toto/pull/368))
+
+## Tangential work
+On a related side-note, together with the TUF team we have made exiting
+progress in in-toto's in-house crypto and utility library -- *securesystemslib*:
+
+### Relevant documents
+
+- A PKCS11-based HSM interface
+  ([#229](https://github.com/secure-systems-lab/securesystemslib/pull/229))
+- nistp384 signature verification support
+  ([#228](https://github.com/secure-systems-lab/securesystemslib/pull/228))
+- A filesystem abstraction for non-local filesystems
+  ([#232](https://github.com/secure-systems-lab/securesystemslib/pull/232))
+
+We plan to adopt some of these features for in-toto in next as part of roadmap
+'21.
+
+
+### Closing remarks.
+
+We acknowledge that we missed our main roadmap '20 goal, that is the release of
+in-toto 1.0. But it should be noted that it is a close miss and we are
+confident about our decision to hold off the release until we have dealt with
+the last remaining blemish (i.e.
+[#80](https://github.com/in-toto/docs/issues/90), so that we can guarantee the
```

Comment:
This is 90 right?

Reply:
Very good eyes. It's #80. Fixed this and the typo above in force push.